relay-dynamic-config/src/metrics.rs

```diff
@@ -21,15 +21,29 @@ pub struct Metrics {
     /// List of patterns for blocking metrics based on their name.
     #[serde(skip_serializing_if = "GlobPatterns::is_empty")]
     pub denied_names: GlobPatterns,
+    /// List of patterns of tags to remove.
```

Comment:
Could you add a bit more documentation on what that does? In contrast to the option above, it is worth pointing out that removing tags doesn't drop the overall metric bucket (i.e. the metric is still queryable without the removed tags).

You can also expand the docs on `TagBlock`, in which case it would be useful to actively refer to the docs on the struct.